Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/osgeo/gdal:ubuntu-small-3.7.3 as gdal
+FROM ghcr.io/osgeo/gdal:ubuntu-full-3.7.3 as gdal
```

Comment:
This image is relay too big to be used by default. If you need that, you should create a separate image!

Reply:
Thanx @SBrunner, 
I agree the full image is too large.
I'm not aware how to build Arrow into GDAL separately (except via conda, is that an option?)
What is your suggestion here?